nextcord/message.py

```diff
@@ -543,6 +545,61 @@ def flatten_handlers(cls):
     return cls
 
 
+class MessageInteraction(Hashable):
+    """Represents a message's interaction data.
+
+    A message's interaction data is a property of a message when the message
+    is a response to an interaction from any bot.
+
+    .. versionadded:: 2.1
```

Comment:
Probably the last suggestion I'll have. These docs can be added due to it being Hashable.

```suggestion
    .. versionadded:: 2.1
    
    .. container:: operations

        .. describe:: x == y

            Checks if two message interactions are equal.

        .. describe:: x != y

            Checks if two interaction messages are not equal.

        .. describe:: hash(x)

            Returns the message interaction's hash.
```